frappe/tests/test_commands.py

```diff
@@ -63,24 +63,40 @@ def clean(value):
 	return value
 
 
-def exists_in_backup(doctypes, file):
+def missing_in_backup(doctypes, file):
 	"""Checks if the list of doctypes exist in the database.sql.gz file supplied
```

Comment:
I feel this should be changed to signify that the function returns a list of missing tables